### ISSUE TITLE
Support standalone version

### ISSUE THREAD

**Issue by ph168** [sazap10/bower-loadgo#1]
There is now a standalone version of LoadGo that doesn't require jQuery: https://github.com/franverona/loadgo/releases/tag/2.0
I'd rather use this version as bower dependency in one of my projects which wouldn't be possible currently.
@sazap10 What do you think? Should we create a new bower package for the standalone version?
Maybe we should also ask the original author about this. Regarding updates I would rather see the bower.json in the original repository…

**Comment by franverona** [comment]
Hello guys,

I have to add a bower.json file to LoadGo repo (my bad!). Both solutions could be great, but I prefer a bower-loadgo package with both (jQuery and pure JS versions).

But feel free to take your best choice. I really appreciate the effort!

-Fran

**Comment by ph168** [comment]
@franverona, thank you for your answer! I thought a bit about this and I'm still not sure how to treat both jQuery and standalone versions with bower. (Maybe we should move this discussion to the loadgo repository now as you already said that you would provide the bower.json file…)

Unfortunately we don't have the ability of having jQuery as optional/extra dependency of the bower package (https://github.com/bower/bower/issues/391, https://github.com/bower/bower/issues/308). I just wouldn't want to have jQuery in my project when I don't need it. Conversely, it would be less-than-ideal to have the jQuery-version in a package that doesn't require jQuery.
In my opinion, these problems could only be solved by having two separate bower packages for the two versions. Or are there any other possibilities?

**Comment by franverona** [comment]
You're absolutely right. If I don't want jQuery in my project, it would be better to just download and use a "non-jquery" version instead of having all files.

I'm going to open an issue on LoadGo repository, so we can discuss it there if you like.

-Fran
